MsCorePkg/MacAddressEmulationDxe/MacAddressEmulationDxe.h

```diff
@@ -0,0 +1,73 @@
+/** @file
+
+  DXE Driver header for handling MAC Address Emulation.
+
+  Copyright (C) Microsoft Corporation.
+  SPDX-License-Identifier: BSD-2-Clause-Patent
+
+**/
+
+#pragma once
+
+#include <Uefi.h>
+#include <Protocol/SimpleNetwork.h>
+
+//
+// Utility macro that splits EFI_MAC_ADDRESS into
+// a set of CHAR8 arguments (e.g. for printing)
+//
+#define MAC_TO_CHAR8_ARGS(MacAddress) \
+  ((UINT8 *)&MacAddress)[0], \
+  ((UINT8 *)&MacAddress)[1], \
+  ((UINT8 *)&MacAddress)[2], \
+  ((UINT8 *)&MacAddress)[3], \
+  ((UINT8 *)&MacAddress)[4], \
+  ((UINT8 *)&MacAddress)[5]
+
+typedef struct {
+  VOID            *Registration;
```

Comment:
can this be typed

Reply:
I believe `EfiNamedEventListen` returns `OUT VOID* Registration`, also good catch seems this wasn't being assigned (or used). Perhaps better to remove entirely?